index.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+      <meta charset="utf-8" />
+      <title>Carrera Programmer - Portfolio</title>
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      <link rel="preconnect" href="https://fonts.googleapis.com">
+      <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+      <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&display=swap" rel="stylesheet">
+      <link rel="stylesheet" href="styles.css">
+      <script src="https://kit.fontawesome.com/321e54ee43.js" crossorigin="anonymous"></script>
+    </head>
+    <body>
+
+    <!-- Here starts the header section -->
+
+      <header>
+        <nav class="navbar">
+            <a class="logo-link"><p class="logo">Carrera<i  id=logo-dat class="fa-solid fa-circle-dot"></i>Programmer</p></a>
+            <i id="burguer-menu" class="fa-solid fa-bars"></i>
+        </nav>
+        <section class="welcome-section">
+            <h1>Hey There<span>.</span><br> I'm Carrera</h1>
+            <h2>I'm a Full Stack Developer</h2>
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don't hestiate to contact me.</p>
+            <ul class="social-icons">
+                <li><a href="https://github.com/carreraprogrammer"><i class="fa-brands fa-github-alt"></i></a></li>
+                <li><a href="https://www.linkedin.com/in/daniel-carrera-paz-85a917244/"><i class="fa-brands fa-linkedin-in"></i></a></li>
+                <li><a href="https://twitter.com/carreraprog"><i class="fa-brands fa-twitter"></i></a></li>
+            </ul>
+            <div class="illustration-header-mobile">
+                <img class="vectors" src="vectors/illustration-header-mobile.png" alt="illustration-header">
+            </div>      
```

Comment:
I kindly recommend you repair the link to your image. It doesn't appear on your website

![Screenshot 2022-08-27 154059](https://user-images.githubusercontent.com/109081622/187047346-f99753b6-7a96-4925-bab1-771728920047.png)